tycho-testing-harness/src/main/java/org/eclipse/tycho/test/util/EnvironmentUtil.java

```diff
@@ -36,7 +36,7 @@ public class EnvironmentUtil {
 
     private static final String MAVEN_HOME_INFO = "Maven home:";
 
-    public static final String ECLIPSE_LATEST = "https:////download.eclipse.org/releases/2022-12/";
+    public static final String ECLIPSE_LATEST = "https:////download.eclipse.org/releases/2023-09/";
```

Comment:
do you know hy we have the four times `/` here? it seems to not harm but looks strange...

Reply:
I would guess 2 // for the protocol and 2 more to escape them somewhere in the code.

Reply:
This looks suspicious... if we need escaping it should not be done in the constant itself ...